guide/07-working-with-bigdata/01-introduction.ipynb

```diff
@@ -180,6 +180,13 @@
    "source": [
```

Comment:
<p>The&nbsp;<code><a href="https://esri.github.io/arcgis-python-api/apidoc/html/arcgis.geoanalytics.toc.html" target="_blank">arcgis.geoanalytics</a></code>&nbsp;module available in ArcGIS API for Python provides submodules, data types, classes, and functions to process your big data using an&nbsp;<a href="http://server.arcgis.com/en/server/latest/get-started/windows/what-is-arcgis-geoanalytics-server-.htm" target="_blank">ArcGIS GeoAnalytics Server</a>. The ArcGIS GeoAnalytics Server enables distributed analysis across multiple ArcGIS Server machines. You can analyze more data in less time because you can harness the compute power of multiple machines. Navigate to the&nbsp;<a href="http://enterprise.arcgis.com/en/server/latest/get-started/windows/configure-the-portal-with-arcgis-geoanalytics-server.htm" target="_blank">Get Started with GeoAnalytics Server</a>&nbsp;page to learn how to configure the GeoAnalytics Server with your ArcGIS Enterprise deployment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>